src/callbacks/callbacks.jl

```diff
@@ -301,3 +301,35 @@ function gc_func(integrator)
     )
     return nothing
 end
+
+"""
+    maybe_graceful_exit(integrator)
+
+This callback is called after every timestep
+to allow users to gracefully exit a running
+simulation. To do so, users can navigate to
+and open `$output_dir/graceful_exit.dat`, change
+the file contents from 0 to 1, and the running
+simulation will gracefully exit with the integrator.
```

Comment:
```suggestion
This callback is called after every timestep
to allow users to gracefully exit a running
simulation. To do so, users can navigate to
and open `$output_dir/graceful_exit.dat`, change
the file contents from 0 to 1, and the running
simulation will gracefully exit with the integrator.

!!! note
    This may not be reliable for MPI jobs.
```